zerver/webhooks/insping/doc.md

```diff
@@ -0,0 +1,48 @@
+Insping (stylized as !nsping) is a simple uptime and performance monitoring
+tool. You'll get to know when the website or service is up and running or down.
+
+{!create-stream.md!}
+
+{!create-bot-construct-url.md!}
+
+{!append-stream-name.md!}
+
+{!append-topic.md!}
+
+## Put incoming webhook to Insping
+
+1. Go to [Insping website](https://www.insping.com). If you haven't created an
+   account, sign up for a free account.
+
+2. Create an organization in the
+   [manage section](https://app.insping.com/#manage)
+   if there isn't one and set an URL that you want to monitor. After it's done,
+   it should look something like this:
+   ![](/static/images/integrations/insping/001.png)
+
```

Comment:
I don't feel like it is worth another point because a default organization is already created when the user signed up. So I think I'll leave it at that.

Reply:
Oh, that makes sense now. Thanks for clarifying. Applied suggestion.